packages/client/test/service.js

```diff
@@ -3,8 +3,34 @@ import { sha256 } from 'multiformats/hashes/sha2'
 import { importCar, importBlob, importDirectory } from './importer.js'
 import { Response, Request } from './mock-server.js'
 import * as CBOR from '@ipld/dag-cbor'
-// @ts-ignore - not tracked by TS
-import { setIn } from '../../api/src/utils/utils.js'
```

Comment:
We need this fix ported from main don't we?

Reply:
ah i just gave up on this, the tsconfig in this package is weird and importing from the api package breaks things.... below you can see the `setIn` copy pasted...